product_pricelist_report_qweb/wizard/product_pricelist.py

```diff
@@ -19,62 +19,60 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ###############################################################################
-from openerp.osv import fields, osv
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
 
 
-class ProductPriceList(osv.osv_memory):
+class ProductPriceList(models.TransientModel):
     _inherit = 'product.price_list'
```

Comment:
Could you use something less generic, please? ANd without `_` in model name

Reply:
Comment not applicable, because the model `product.price_list` is native from Odoo, it's just being inherited, so resolving.